src/components/apiproduct/APIProductForm.tsx

```
@@ -0,0 +1,487 @@
+  };
+
+  const handleApprovalModeChange = (approvalMode: 'automatic' | 'manual') => {
+    onFormDataChange({ ...formData, approvalMode });
+  };
+
+  return (
+    <Card style={{ maxWidth: '960px' }}>
+      <CardBody>
+        <Form>
+          {/* API product info section */}
+      <Title
```

ESLint

[error] Insert `····`

(prettier/prettier)